test/product.test.js

```diff
@@ -6,4 +6,8 @@ test('multiplies 1 * 2 to equal 2', () => {
 
 test('multiplies 3 * 4 to equal 12', () => {
   expect(product(3, 4)).toBe(12);
+});
+
+test('multiplies 1.5 * 1.5 to equal 12', () => {
```

Comment:
Everything looks great! If you don't mind fixing this test case "to equal 2.25" I'll merge away. Sometimes C & P gets the best of us.